.github/workflows/inspektor-gadget.yml

```diff
@@ -22,12 +22,28 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v1
 
-    - name: Install dependencies
+    - name: Cache deb packages
+      uses: actions/cache@v2
+      id: cache-debs
+      with:
+          path: "~/cache-debs"
+          key: cache-debs-libbpf-1:0.4.0-1ubuntu1
+
+    - name: Install deb packages
+      env:
+        CACHE_HIT: ${{steps.cache-debs.outputs.cache-hit}}
+        LIBBPF_VERSION: "1:0.4.0-1ubuntu1"
```

Comment:
What would happen a new version is released? Should we update manually this reference?

Reply:
Yes, it will not automatically switch to the new version. Note that there are two references with the version: line 30 and line 35. I was not sure how to use environment variables to keep it in sync.